roachtest: gorm failed

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3641256&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3641256&tab=artifacts#/gorm) on master @ [965b99b1a05eb893ee908d852680824a0bd4ffe3](https://github.com/cockroachdb/cockroach/commits/965b99b1a05eb893ee908d852680824a0bd4ffe3):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gorm/run_1
	orm_helpers.go:245,orm_helpers.go:171,java_helpers.go:220,gorm.go:132,test_runner.go:777: 
		Tests run on Cockroach v22.1.0-alpha.00000000-711-g965b99b1a0
		Tests run against gorm v1.22.0
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #72022 roachtest: gorm failed [C-test-failure O-roachtest O-robot branch-release-21.2 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=artifacts#/gorm) on master @ [db229cad354c2776ea35c9dc4f78e6ce66b9437c](https://github.com/cockroachdb/cockroach/commits/db229cad354c2776ea35c9dc4f78e6ce66b9437c):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gorm/run_1
	orm_helpers.go:245,orm_helpers.go:171,java_helpers.go:220,gorm.go:132,test_runner.go:777: 
		Tests run on Cockroach v22.1.0-alpha.00000000-827-gdb229cad35
		Tests run against gorm v1.22.2
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #72022 roachtest: gorm failed [C-test-failure O-roachtest O-robot branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3661274&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3661274&tab=artifacts#/gorm) on master @ [a15b840db34de504bf060b2599276e049cc64799](https://github.com/cockroachdb/cockroach/commits/a15b840db34de504bf060b2599276e049cc64799):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gorm/run_1
	orm_helpers.go:245,orm_helpers.go:171,java_helpers.go:220,gorm.go:132,test_runner.go:777: 
		Tests run on Cockroach v22.1.0-alpha.00000000-834-ga15b840db3-dirty
		Tests run against gorm v1.22.2
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #72022 roachtest: gorm failed [C-test-failure O-roachtest O-robot branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667494&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667494&tab=artifacts#/gorm) on master @ [f69253e3d1cfe798821ea1b1c85aeb79e9eece32](https://github.com/cockroachdb/cockroach/commits/f69253e3d1cfe798821ea1b1c85aeb79e9eece32):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gorm/run_1
	orm_helpers.go:245,orm_helpers.go:171,java_helpers.go:220,gorm.go:132,test_runner.go:777: 
		Tests run on Cockroach v22.1.0-alpha.00000000-864-gf69253e3d1
		Tests run against gorm v1.22.2
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #72022 roachtest: gorm failed [C-test-failure O-roachtest O-robot branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674519&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674519&tab=artifacts#/gorm) on master @ [05d36621ae805e644d98ef6b35d453ae16255ccc](https://github.com/cockroachdb/cockroach/commits/05d36621ae805e644d98ef6b35d453ae16255ccc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gorm/run_1
	orm_helpers.go:245,orm_helpers.go:171,java_helpers.go:220,gorm.go:132,test_runner.go:777: 
		Tests run on Cockroach v22.1.0-alpha.00000000-899-g05d36621ae
		Tests run against gorm v1.22.2
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #72022 roachtest: gorm failed [C-test-failure O-roachtest O-robot branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/gorm.go
+++ b/pkg/cmd/roachtest/tests/gorm.go
@@ -22,7 +22,7 @@ import (
 )
 
 var gormReleaseTag = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var gormSupportedTag = "v1.21.12"
+var gormSupportedTag = "v1.22.2"
 
 func registerGORM(r registry.Registry) {
 	runGORM := func(ctx context.Context, t test.Test, c cluster.Cluster) {
@@ -111,7 +111,7 @@ func registerGORM(r registry.Registry) {
 		err = c.RunE(
 			ctx,
 			node,
-			fmt.Sprintf(`cd %s && go get -t -u ./... && go mod download && go mod tidy`, gormTestPath),
+			fmt.Sprintf(`cd %s && go get -u -t ./... && go mod download && go mod tidy `, gormTestPath),
 		)
 		require.NoError(t, err)
 
@@ -121,8 +121,8 @@ func registerGORM(r registry.Registry) {
 		err = c.RunE(
 			ctx,
 			node,
-			fmt.Sprintf(`cd %s && GORM_DIALECT="postgres" 
-PGUSER=root PGPORT=26257 PGSSLMODE=disable go test -v ./... 2>&1 | %s/bin/go-junit-report > %s`,
+			fmt.Sprintf(`cd %s && GORM_DIALECT="postgres"
+		PGUSER=root PGPORT=26257 PGSSLMODE=disable go test -v ./... 2>&1 | %s/bin/go-junit-report > %s`,
 				gormTestPath, goPath, resultsPath),
 		)
 		if err != nil {

